packages/legacy/core/App/container-api.ts

```diff
@@ -19,12 +19,24 @@ import { NotificationReturnType, NotificationsInputProps } from './hooks/notific
 import { NotificationListItemProps } from './components/listItems/NotificationListItem'
 import { PINCreateHeaderProps } from './components/misc/PINCreateHeader'
 import { CredentialListFooterProps } from './types/credential-list-footer'
+import { ViewStyle } from 'react-native'
 
 export type FN_ONBOARDING_DONE = (
   dispatch: React.Dispatch<ReducerAction<unknown>>,
   navigation: StackNavigationProp<AuthenticateStackParams>
 ) => GenericFn
 
+export enum InlineErrorPosition {
+  Above,
+  Below,
+}
+
+export type InlineErrorConfig = {
+  enabled: boolean
+  position?: InlineErrorPosition
+  style?: ViewStyle
+}
+
```

Comment:
It might be better for these to be in `types.ts` or `InlineErrorText.tsx` to keep things close to what they are most related to and keep `container-api.ts` focused. I know there are a couple existing type definitions in this file already but I think that was a misstep on our part, I can move those in a future PR.